clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -813,6 +813,7 @@ public KafkaConsumer(Map<String, Object> configs,
             this.kafkaConsumerMetrics = new KafkaConsumerMetrics(metrics, metricGrpPrefix);
 
             config.logUnused();
+            config.logUnknown();
```

Comment:
It's a little weird and easy to forget to call these 2 methods after client initialization. Could we create a `public logUnknownAndUnused` method, and make both `logUnused` and `logUnknown` as private, so that users will always call `logUnknownAndUnused` to log both. WDYT?

Reply:
Sure.